app/helpers/bike_share_app_helper.rb

```diff
@@ -0,0 +1,10 @@
+module BikeShareAppHelper
+
+  def format_date(date)
```

Comment:
this method looks really familiar. i like that you've extracted this out to a method